pallets/common/src/traits/settlement.rs

```diff
@@ -83,6 +84,9 @@ decl_event!(
         /// An instruction affirmation has been withdrawn by a mediator.
         /// Parameters: [`IdentityId`] of the mediator and [`InstructionId`] of the instruction.
         MediatorAffirmationWithdrawn(IdentityId, InstructionId),
+        /// An instruction with mediators has been created.
+        /// Parameters: [`InstructionId`] of the instruction and the [`IdentityId`] of all mediators.
+        InstructionMediators(InstructionId, BTreeSet<IdentityId>),
```

Comment:
(nit) I thought events were supposed to be past tense.

I am not sure on a better name myself though 😅

Maybe something like `InstructionMediatorsRequired` ?

Reply:
I don't think there is a rule to use past tense.  We just kind of did it that way.  Also I think the current name is fine.  Unless some better name is found.  Not a fan of `InstructionMediatorsRequired`.

Reply:
I'm ok with that change, specially if it makes more sense for you. @Neopallium any thoughts on this?

Reply:
Maybe `InstructionMediatorsAdded` is a bit better?

Reply:
I don't think "Added" makes sense.  Maybe `InstructionCreatedWithMediators`, if we really want to have past tense.  Or `InstructionCreatedExtension`, since it is an extension of the `InstructionCreated` event.

Reply:
Lets leave it as it is then